bespin-web/scripts/start-apache.sh

```diff
@@ -5,7 +5,13 @@ python manage.py migrate
 
 create-lando-user.sh
 create-lando-connection.sh
-load-sample-data.sh
+LOADEDSAMPLEDATA=/var/run/bespin-load.txt
```

Comment:
The problem is every time you restart bespin-web it loads the fixtures and resets the DukeDS settings. So you have to go in and recreate them. Perhaps removing the two DukeDS them from the fixtures entirely would remedy this.